contracts/token/ERC20/extensions/ERC4626.sol

```diff
@@ -153,19 +162,43 @@ abstract contract ERC4626 is ERC20, IERC4626 {
         uint256 supply = totalSupply();
         return
             (assets == 0 || supply == 0)
-                ? assets.mulDiv(10**decimals(), 10**_asset.decimals(), rounding)
+                ? _initialConvertToShares(assets, rounding)
                 : assets.mulDiv(supply, totalAssets(), rounding);
     }
 
+    /**
+     * @dev Internal conversion function (from assets to shares) to apply when the vault is empty.
+     */
+    function _initialConvertToShares(uint256 assets, Math.Rounding rounding)
+        internal
+        view
+        virtual
+        returns (uint256 shares)
+    {
+        rounding;
```

Comment:
Is there a compiler error if we remove the name from the argument list?

Reply:
no, but then people might miss what the argument is about.

What do you feel about 
```
function _initialConvertToShares(uint256 assets, Math.Rounding /* rounding */)
```
?